src/main/java/cn/nukkit/entity/mob/EntitySnowGolem.java

```diff
@@ -0,0 +1,17 @@
+package cn.nukkit.entity.mob;
+
+import cn.nukkit.level.format.FullChunk;
+import cn.nukkit.nbt.tag.CompoundTag;
+
+public class EntitySnowGolem extends EntityMob {
+    public static final int NETWORK_ID = 21;
+
+    @Override
+    public int getNetworkId() {
+        return NETWORK_ID;
+    }
+
+    public EntitySnowGolem(FullChunk chunk, CompoundTag nbt) {
+        super(chunk, nbt);
+    }
```

Comment:
I think you should include at least the minimal information such as width, height, name and health

Reply:
👍